src/mem3/src/mem3_shards.erl

```diff
@@ -210,12 +216,25 @@ handle_cast({cache_hit, DbName}, St) ->
     couch_stats:increment_counter([mem3, shard_cache, hit]),
     cache_hit(DbName),
     {noreply, St};
-handle_cast({cache_insert, DbName, Shards}, St) ->
-    couch_stats:increment_counter([mem3, shard_cache, miss]),
-    {noreply, cache_free(cache_insert(St, DbName, Shards))};
+handle_cast({cache_insert, DbName, Writer, UpdateSeq}, St) ->
+    NewSt = case UpdateSeq < St#st.update_seq of
```

Comment:
Nope. Took me a bit of noodling over this question. In the end its easy enough to see that it has to be < for when the dbs db isn't chaning. At steady state if it were =< then nothing would ever be inserted into the cache.